manifests/cpp.yml

```diff
@@ -241,8 +241,14 @@ tests/:
     Test_Config_HttpServerErrorStatuses_FeatureFlagCustom: missing_feature
     Test_Config_IntegrationEnabled_False: missing_feature
     Test_Config_IntegrationEnabled_True: missing_feature
+    Test_Config_LogInjection_128Bit_TradeId_Default: missing_feature
```

Comment:
Instead of missing feature can we use https://github.com/DataDog/system-tests/blob/25c79a3502d69e30d8ccf0461d15501ccbd0addf/docs/edit/skip-tests.md?plain=1#L5. Some of these fail because the the parametric apps do not implement the new endpoint.